recipes/cerberecontrib_ghrsst/meta.yaml

```diff
@@ -0,0 +1,49 @@
+
+{% set name = "cerberecontrib-ghrsst" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.ifremer.fr/cerbere/cerberecontrib-ghrsst/-/archive/{{ version }}/cerberecontrib-ghrsst-{{ version }}.tar.gz
+  sha256: 7f84aeb91bb181c20b2ebe9f9405407d0f4717a187e66f4366eb4ad49cd970c3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10,<3.11
+    - poetry-core >=1.0.0
+    - poetry-dynamic-versioning >=1.0.0,<2.0.0
+    - pip
+  run:
+    - python >=3.10.0,<3.11
+    - cerbere >=3.1.2
+
+test:
```

Comment:
So your test is failing. 
`cerberecontrib-ghrsst 0.0.0.post146767+30809b91c0 has requirement cerbere<4.0.0,>=3.1.2, but you have cerbere 0.0.0.post21+c94c7d1.`
So I'd say maybe from this we need to add in the test the import of cerbere